### PR TITLE
feat: release pipeline

### PR DESCRIPTION
**Pull request by berendsliedrecht**
Signed-off-by: Berend Sliedrecht <sliedrecht@berend.io>

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -49,3 +49,42 @@ jobs:
...
+          - name: Publish
+            id: publish
+            env:
+              NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            run: |
+              npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN
+
+              if [ ${{ inputs.release-type }} == 'alpha' ]; then
+                pnpm release --preRelease=alpha
```
Comment (TimoGlastra) [inline .github/workflows/ci.yaml:87]: do we also need to provide the release type here? E.g. if we're on 1.0.0 now, if we do alpha release should it be 1.0.1-alpha.0, 1.1.0-alpha.0, or 2.0.0-alpha.0?

File: .release-it.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "git": {
+    "commitMessage": "chore: release v${version}"
```
Comment (TimoGlastra) [inline .release-it.json:3]: If it makes a commit on release, we should also push that commit. But I don't see support for that in the current CI setup.

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "askar-webcrypto",
+  "name": "@animo-id/askar-webcrypto",
   "version": "0.0.0",
   "main": "build/index.js",
-  "author": "Animo Solutions <development@animo.id> (https://animo.id)",
+  "author": "Animo Solutions <development@animo.id> (https://www.animo.id)",
```
Comment (TimoGlastra) [inline package.json:5]: Woot. https://www.animo.id doesn't seem to work. I'll look into into it, but why not just https://animo.id?

File: .release-it.json (hunk 2 above)
Comment (TimoGlastra) [reply .release-it.json:3]: We  can probably give the CI push access. See https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

**Review by TimoGlastra — APPROVED** [review]

File: .release-it.json (hunk 2 above)
Comment (TimoGlastra) [reply .release-it.json:3]: But if we enable branch protection it won't be possible anymore. So we either need to take the approach of not commiting when we make a release (as we do in more libraries, by not looking at package.json for the next version), or we need to disable brancn protectino and allow pushing to main, or we need to make a PR first to increase the version, and based on that trigger a release.

I'm fine with a simple solution for now

File: package.json (hunk 3 above)
Comment (berendsliedrecht) [reply package.json:5]: lol thats weird, we are not on the world wide web!!!!

https://animo.id works as well, I prefer the looks of `www` haha.

File: .release-it.json (hunk 2 above)
Comment (berendsliedrecht) [reply .release-it.json:3]: Good catch!

**Pull request merged**